app/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,9 @@
     <string name="visitable_recycler_adapter_unregistered_message">
         Resource %1$s.xml not registered on VisitableRecyclerAdapter!
     </string>
+    <string name="type_city_province_region">Type city or province region ...</string>
```

Comment:
This can be improved, can you change into 
Type country or province region...

Reply:
noted,👍